[DOCS-421] DogStatsD standalone package (#6455)
DataDog/documentation: content/en/agent/_index.md

The Datadog Agent is software that runs on your hosts. It collects events and metrics from hosts and sends them to Datadog, where you can analyze your monitoring and performance data. The Datadog Agent is open-source, and its source code is available on GitHub at [DataDog/datadog-agent][1].

> **It is recommended to fully install the Agent.** However, a standalone DogStatsD package is available for Amazon Linux, CentOS, Debian, Fedora, Red Hat, SUSE and Ubuntu. This package is for containerized environments where DogStatsD needs to be run as a sidecar or for those who need to install a DogStatsD server without full Agent functionality.

Reviewer: ```suggestion
**It is recommended to fully install the Agent.** However, a standalone DogStatsD package is available for Amazon Linux, CentOS, Debian, Fedora, Red Hat, SUSE and Ubuntu. This package is used in containerized environments where DogStatsD runs as a sidecar or environments running a DogStatsD server without full Agent functionality.
```

> The standalone DogStatsD package is installed exactly as the full agent version (see an example [here][2]) **except** that every occurrence of `Datadog-agent` should be replaced with `Datadog-dogstatsd`. A docker image is available in the [DogStatsD6 docker image repo][3].

Reviewer: ```suggestion
The standalone DogStatsD package is installed with the Agent [one-line install command][2] **except** every occurrence of `datadog-agent` should be replaced with `datadog-dogstatsd`. A Docker image is available in the [DogStatsD6 Docker image repo][3].
```
